config/samples/install_v1alpha1_lookout.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: install.armadaproject.io/v1alpha1
+kind: Lookout
+metadata:
+  labels:
+    app.kubernetes.io/name: lookout
+    app.kubernetes.io/instance: lookout-sample
+    app.kubernetes.io/part-of: armada-operator
+    app.kubernetes.io/created-by: armada-operator
+  name: lookout-sample
+  namespace: default
+spec:
+  image:
+    repository: gresearchdev/armada-lookout
+    tag: v0.3.44
+  applicationConfig:
```

Comment:
It may be worth updating this with a similar dev config that I sent out.  I think right now you are using the executor config which is not very useful for lookout.